asv/plugins/__init__.py

```diff
@@ -1 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# Import to make sure plugins are discoverable
+import asv.plugins.conda
+import asv.plugins.mamba
```

Comment:
This shouldn't be required? (plugins are auto-discovered, failures mean they are dropped from valid options)